portal-ui/src/screens/LoginPage/LoginPage.tsx

```diff
@@ -86,42 +66,118 @@ const styles = (theme: Theme) =>
       justifyContent: "center",
       alignItems: "center",
       position: "absolute",
+      zIndex: 9,
       left: "50%",
       top: "50%",
-      marginLeft: -400,
-      marginTop: -290,
+
       color: "#fff",
       fontWeight: 700,
       fontSize: 14,
       borderRadius: 8,
       padding: 10,
       boxSizing: "border-box",
     },
-    mainContainer: {
+    loginPage: {
+      height: "100%",
+      display: "flex",
+      flexFlow: "column",
+      alignItems: "stretch",
       position: "relative",
-      height: 424,
+      padding: 84,
+
+      "@media (max-width: 900px)": {
+        padding: 0,
+      },
     },
-    theOcean: {
-      borderTopLeftRadius: 8,
-      borderBottomLeftRadius: 8,
-      background:
-        "transparent linear-gradient(to bottom, #073052 0%,#05122b 100%); 0% 0% no-repeat padding-box;",
+    shadowBox: {
+      boxShadow: "0px 3px 10px #00000014",
+      height: "100%",
     },
-    oceanBg: {
-      backgroundImage: "url(/images/BG_Illustration.svg)",
-      backgroundRepeat: "no-repeat",
-      backgroundPosition: "bottom left",
+    loginContainer: {
+      flex: 1,
       height: "100%",
-      width: 324,
+
+      "& .right-items": {
+        padding: 50,
+        flex: 1,
+        height: "100%",
+        display: "flex",
+        flexFlow: "column",
+        alignItems: "center",
+        justifyContent: "center",
+        maxWidth: "33%",
+
+        "@media (max-width: 900px)": {
+          maxWidth: "100%",
+          margin: "auto",
+        },
+      },
+      "& .consoleTextBanner": {
+        // font: "normal normal 100 100px/100px Lato",
+        fontFamily: "Lato, sans-serif",
```

Comment:
Can we make this font thinner?
```suggestion
        fontWeight: "300",
```